src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp

```diff
@@ -651,6 +660,31 @@ namespace TerminalAppLocalTests
             VERIFY_ARE_EQUAL(til::color(myArgs.TerminalArgs().TabColor().Value()), expectedColor);
             VERIFY_IS_NULL(myArgs.TerminalArgs().ProfileIndex());
             VERIFY_IS_TRUE(myArgs.TerminalArgs().Profile().empty());
+            VERIFY_IS_TRUE(myArgs.TerminalArgs().ColorScheme().empty());
+        }
+        {
+            AppCommandlineArgs appArgs{};
+            std::vector<const wchar_t*> rawCommands{ L"wt.exe", subcommand, L"--colorScheme", L"Monokai" };

```

Comment:
Looks like you've hard coded "Monokai" for the color scheme. This should be a empty string looking for input from the user.

Reply:
It's a test, of course I hardcoded the string...